README.md

```diff
@@ -1,9 +1,15 @@
-# rentreceipt
+# Rent Receipt Generator
 Rent Receipt for Employees in India
 
 To create your Rent Receipt pdf file:
 
-1. Install LaTeX.
-2. Modify the rentreceipt.tex and execute:
+### Prerequisites
```

Comment:
Could you please specify the distro to be used for executing the commands, as the latex package names differ for different distributions ? For instance, in debian, it may be different.